app/src/main/java/org/mozilla/focus/fragment/UrlInputFragment.kt

```diff
@@ -246,6 +246,9 @@ class UrlInputFragment :
             if (!Settings.getInstance(it).shouldShowFirstrun()) {
                 // Only show keyboard if we are not displaying the first run tour on top.
                 showKeyboard()
+            } else {
+                // If we are showing first run tour, make sure accessibility is turned off.
+                view?.rootView?.importantForAccessibility = View.IMPORTANT_FOR_ACCESSIBILITY_NO_HIDE_DESCENDANTS
             }
```

Comment:
I wonder if we could temporarily detach the other fragment (browser or input) and then reattach it once first run is done:
https://developer.android.com/reference/android/app/FragmentTransaction.html#detach(android.app.Fragment)